src/styles/_catalog.scss

```diff
@@ -1,22 +1,25 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$card-width: 200px;
+
+@mixin col-width($width, $col) {
+  @media (min-width: $width) {
+    grid-template-columns: repeat($col, $card-width);
+  }
+}
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  @include col-width(488px, 2);
+  @include col-width(768px, 3);
+  @include col-width(1024px, 4);
```

Comment:
that's what i see on <488px, you should fix it
![image](https://user-images.githubusercontent.com/37161315/169849002-9ddf9580-aef4-481a-851f-429b91157701.png)